cpp/ps0003/p03.cpp

```diff
@@ -4,11 +4,17 @@ using namespace std;
 
 int main()
 {
-  int x=0;
-  cin>>x;
- if (x%2==0)
-    cout<< "Ever"<<endl;
- else
-     cout<< "Odd"<<endl;
+  // prompting user for number 2/2
+  int x = 0;
+  cin >> num;
 
+  // checking if the number is odd or even 3/3
+  cout << "Number is ";
+  if (num % 2 == 0)
+    cout << "even" << endl;
+  else
+    cout << "odd" << endl;
+  
+  return 0;
 }
+// great work 😎 8/10 ⭐

```

Comment:
- [ ] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
you can visit the [problem solution](https://onlinegdb.com/PbqxNbSVc) for more

you also can use _ternary operator_:
```cpp
std::cout << "Number is " << ((number % 2 == 0) ? "even" : "odd") << std::endl
```